pkg/agent/controller/supportbundlecollection/support_bundle_controller.go

```diff
@@ -0,0 +1,439 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package supportbundlecollection
```

Comment:
Could you place it under path /pkg/agent/supportbundlecollection directly? Since all related logic is in the same path, and it is not about the controlplane/dataplane implementation in the feature, a separate path is more appropriate.

Reply:
Sure, have updated accordingly.